server/src/resolvers/roomsResolver.ts

```diff
@@ -63,8 +63,18 @@ const RoomResolvers = {
       const room = await RoomRepo.getRoomByID(args.roomID);
       assert(room);
 
-      const hashedUniversityID = hashUniversityID(args.universityID);
-      const user = await UserRepo.getUserByUniversityID(hashedUniversityID);
+      let hashedUIDString = "";
+
+      const users = await UserRepo.getUsers();
+      users.forEach(user => {
```

Comment:
Why are we looping through every user in the system? Can't we just grab the hashed UID from the current user using the callback argument in `ifAllowed`?

Reply:
will the current User always be the swiped in user? If so, would makes sense with that route